<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyProjectsPage</class>
 <widget class="QWidget" name="MyProjectsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="hint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Double-click to edit a project</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="projects">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeView {
	show-decoration-selected: 1;
}
QTreeView::item {
	padding:3px 0px; 
	border: 1px solid palette(base);
}
QTreeView::text {
	margin: 0px; 
	padding:0px; 
	subcontrol-origin: border; 
	subcontrol-position: bottom center; 
	border: none;
}
QTreeView::item:selected:active,
QTreeView::item:selected:!active {
	border: 1px solid palette(highlight); 
	background-color: palette(highlight); 
	color: palette(highlighted-text);
}
QTreeView::item:hover {
	border: 1px groove palette(dark); 
	background-color: palette(mid); 
	color: palette(highlighted-text);
}
QTreeView::item:selected:first,
QTreeView::item:hover:first{
	border-left: none; 
	border-right: none;
}
QTreeView::item:selected:last,
QTreeView::item:hover:last{
	border-left: none;
}
QTreeView::branch {
	border:none; 
	image:none; 
	background-color:palette(base); 
	margin:0px; 
	padding-top:2px;
}
QTreeView::branch:selected:active,
QTreeView::branch:selected:!active {
	background-color: palette(highlight); 
	border-right: none;
}
QTreeView::branch:hover {
	border: none; 
/*1px groove palette(dark); */
	background-color: palette(mid); 
	color: palette(highlighted-text); 
	border-right: none;
}
QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
	image: url(:/treeview/expand.png);
}
QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings {
	image: url(:/treeview/collapse.png);
}
</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Project</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modified</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonsLayout" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="linkProjects">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Link project(s)</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/link-projects.png</normaloff>:/icons/link-projects.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportProjects">
        <property name="text">
         <string>Export project(s)</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/export-projects.png</normaloff>:/icons/export-projects.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="syncProjects">
        <property name="text">
         <string>Update project(s)</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/sync-projects.png</normaloff>:/icons/sync-projects.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteProjects">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove project(s)</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/delete-projects.png</normaloff>:/icons/delete-projects.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
